### ISSUE TITLE
Can Admin create new users?

### ISSUE THREAD

**Issue by codeonion** [codeonion/BlogIgniter#2]
Is it a feature to be developed at this or any future stage?

**Comment by imran300** [comment]
Yes he can create users or we can say like if a user register him self so his request for registration will be pending until admin approves it..

**Comment by codeonion** [comment]
I see! That we will do then.
